models/db/convert.go

```diff
@@ -39,3 +42,16 @@ func ConvertUtf8ToUtf8mb4() error {
 
 	return nil
 }
+
+// Cell2Int64 converts a xorm.Cell type to int64,
+// and handles possible irregular cases.
+func Cell2Int64(val xorm.Cell) int64 {
+	switch (*val).(type) {
+	case []uint8:
+		log.Trace("Cell2Int64 ([]uint8): %v", *val)
+
+		v, _ := strconv.ParseInt(string((*val).([]uint8)), 10, 64)
```

Comment:
should error really be ignored?

Reply:
Just move from `models`, haven't take a look at it. :)

Reply:
It could be ignored, even if the error catch, it has to return zero.